### ISSUE TITLE
reCaptcha broken (google api rules)

### ISSUE THREAD

**Issue by jondye** [EFForg/https-everywhere#1682]
When accessing http://epetition.cambridgeshire.public-i.tv/epetition_core/view/miltonrdcycleways the reCaptcha challenge fails to load with a 404 when "Google APIs" rule is enabled. The URL accessed is http://api.recaptcha.net/recaptcha/api/js/recaptcha_ajax.js which is re-written to https://www.google.com/recaptcha/api/recaptcha/api/js/recaptcha_ajax.js which returns a 404.

**Comment by reedy** [comment]
```
	<rule from="^http://api\.recaptcha\.net/"
		to="https://www.google.com/recaptcha/api/" />

	<rule from="^http://api-secure\.recaptcha\.net/"
		to="https://api-secure.recaptcha.net/" />
```

Looks weird that it rewrites this 2 different ways.

Plus visiting the api-secure gives a ssl cert error too, so that's also wrong

This server could not prove that it is api-secure.recaptcha.net; its security certificate is from www.google.com. This may be caused by a misconfiguration or an attacker intercepting your connection.

**Comment by jsha** [comment]
Per https://groups.google.com/forum/#!topic/recaptcha/V7qswqBnA1o, the rewritten URL should be https://www.google.com/recaptcha/api/js/recaptcha_ajax.js.

**Comment by reedy** [comment]
So it's just the duplicate recaptcha/api that needs removing

**Comment by jsha** [comment]
That needs removing. But also, the first rule should capture more of the path: from `api.recaptcha.net/recaptcha/api/` to `www.google.com/recaptcha/api/`

**Comment by RemakingEden** [comment]
@J0WI just looking at this now, Ive check out the new Recaptcha and they all seem to be https automatically. E.g. https://www.google.com/recaptcha/api2/anchor?ar=1&k=6Ld2sf4SAAAAAKSgzs0Q13IZhY02Pyo31S2jgOB5&co=aHR0cHM6Ly9wYXRyaWNraGxhdWtlLmdpdGh1Yi5pbzo0NDM.&hl=en&v=v1548052318968&size=normal&cb=rpnamyfxgsxg

Would you recommend still writing a rule for this? Or should this issue be closed?

I'm struggling to access any site called recaptcha.net or see any reference to it in the embedded recaptcha I have found.

**Comment by pipboy96** [comment]
@J0WI `recaptcha.net` is blocked in China.

**Comment by pipboy96** [comment]
Closed by #17361.
